core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -441,7 +441,7 @@ protected function row($strPalette=null)
 			$key = (Input::get('act') == 'editAll') ? 'FORM_FIELDS_' . $suffix : 'FORM_FIELDS';
 
 			// Calculate the current palette
-			$postPaletteFields = implode(',', Input::post($key));
+			$postPaletteFields = implode(',', (array) Input::post($key));
```

Comment:
How can this ever not be an array?

Reply:
With `overrideAll` this could be `null` I think, but I didn’t test it.

Reply:
The stack trace we received show the error and I think the underlying framework should handle user inputs without crashing. We've a dozen of fixes like these and are willing to contribute these back.

<img width="984" alt="image" src="https://github.com/contao/contao/assets/307605/35fbf94f-6c37-4d45-8eab-f231fd55de2b">